### DISCUSSION TITLE
模型资源 | OhMyLive2D

### DISCUSSION THREAD

**Discussion by giscus** [oh-my-live2d/oh-my-live2d#30] [Announcements]
# 模型资源 | OhMyLive2D

支持所有Live2D模型版本的Web组件, 快速为您的个人网站加入看板娘

https://oml2d.com/guide/models.html

**Comment by xiatianYa** [comment]
酷,希望早日加个碧蓝档案人物的模型

**Reply by KISGP** [reply]
你可以看看这个仓库[https://github.com/Eikanya/Live2d-model](https://github.com/Eikanya/Live2d-model)，里面就有碧蓝的模型

**Reply by qzm4228** [reply]
这不是碧蓝航线吗💀

**Comment by Catho168** [comment]
path能否填入本地路径？如果可以就方便很多了
